### PR TITLE
Y-board buttons and switches outline

### PR DESCRIPTION
**Pull request by meyer-vict2020**

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+build_flags =   
+     -DARDUINO_USB_MODE=1
+     -DARDUINO_USB_CDC_ON_BOOT=1
+     -I src/lib-y-badge
+
+
+lib_deps = 
+     https://github.com/y-board/y-board-v3
+     adafruit/Adafruit NeoPixel
```
Comment (philipbl) [inline platformio.ini]: Can you remove the NeoPixel. The y-board-v3 library should already include that.

Comment (meyer-vict2020) [reply platformio.ini]: Ok.

File: platformio.ini [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+[env:esp32]
+platform = espressif32
+board = esp32-s3-devkitc-1
+framework = arduino
+
+build_flags =   
+     -DARDUINO_USB_MODE=1
+     -DARDUINO_USB_CDC_ON_BOOT=1
+     -I src/lib-y-badge
```
Comment (philipbl) [inline platformio.ini]: Sorry, I missed one more line. You shouldn't need `-I src/lib-y-badge` either. Other than that, the PR looks good.

Comment (meyer-vict2020) [reply platformio.ini]: fixed

**Comment by philipbl** [comment]
Great job!

**Pull request merged**